data/presets/fields/address.json

```diff
@@ -0,0 +1,64 @@
+{
+    "type": "address",
+    "keys": [
+        "addr:block_number",
+        "addr:city",
+        "addr:block_number",
+        "addr:conscriptionnumber",
+        "addr:county",
+        "addr:country",
+        "addr:county",
+        "addr:district",
+        "addr:floor",
+        "addr:hamlet",
+        "addr:housename",
+        "addr:housenumber",
+        "addr:neighbourhood",
+        "addr:place",
+        "addr:postcode",
+        "addr:province",
+        "addr:quarter",
+        "addr:state",
+        "addr:street",
+        "addr:subdistrict",
+        "addr:suburb"
+    ],
+    "reference": {"key": "addr"},
+    "icon": "address",
+    "universal": true,
+    "label": "Address",
+    "strings": {
```

Comment:
The localization module I added yesterday expects an object like this for translations:

```js
{
  en: {
    example: 'example'
  },
  es: {
    example: 'exemplo'
  }
}
```

It would be convenient to use that format here.

I'm not sure, but I suspect that breaking the languages out into separate objects would make it easier to see what's been translated, too.